src/swig/mlt.i

```diff
@@ -18,13 +18,68 @@
  */
 
 %module mlt
+%define MLT_EXPORT
```

Comment:
Thanks for explaining this change to me! Can we add a short code comment to explain this in case someone wonders in the future?